### PR TITLE
Events section

### PR DESCRIPTION
**Pull request by alexanderzobnin**
Additional section for displaying event markers.
**Work in progress, just for review!**

### CODE REVIEW

File: pkg/api/annotations.go [hunk 1]
```diff
@@ -30,17 +33,32 @@ func GetAnnotations(c *middleware.Context) Response {
 	result := make([]dtos.Annotation, 0)
 
 	for _, item := range items {
+		// Get user info for annotation event
+		userName := ""
+		if item.UserId != 0 {
+			userQuery := &models.GetUserByIdQuery{
```
Comment (torkelo) [inline pkg/api/annotations.go]: this looks really bad, as it will issue one query PER annotation :) Think we need to join based on userId

**Comment by torkelo** [comment]
closing this and opening a new PR based on create-annotations branch, please push updates to this grafana branch

File: pkg/api/annotations.go (hunk 1 above)
Comment (alexanderzobnin) [inline pkg/api/annotations.go]: Yea! Just a proof of concept.

Comment (torkelo) [inline pkg/api/annotations.go]: I figured as much, will try to figure out how we should do this. need a read model that is different then the write model

**Pull request closed without merging**